src/main/java/com/lina/airline/service/impl/UserServiceImpl.java

```diff
@@ -27,6 +30,10 @@ public UUID saveUser(UserRegistrationRequest userRegistrationRequest) {
         userEntity.setEmail(userRegistrationRequest.getEmail());
         userEntity.setPassword(userRegistrationRequest.getPassword());
         userEntity.setName(userRegistrationRequest.getName());
+        userEntity.setPhoneNumber(userRegistrationRequest.getPhoneNumber());
```

Comment:
Do we need to add this properties at there? it is a user registration API. Why you are adding it at there